server/server.js

```diff
@@ -5,30 +5,12 @@ const path = require('path')
 const nodeEnv = process.env.NODE_ENV && process.env.NODE_ENV.toLowerCase()
 if (nodeEnv === 'development' || nodeEnv === 'dev' || !nodeEnv) {
   require('dotenv').config()
```

Comment:
Any reason why this is still conditional?

Reply:
No reason other than that is was there before. I don't think it's needed at all as we have an (unconditional) `require('dotenv').config()` in `app.js`. But I ended up deciding that this cleanup was out-of-scope for this task.